commands/publish/command.js

```diff
@@ -40,18 +40,15 @@ exports.builder = yargs => {
       type: "string",
       requiresArg: true,
     },
-    "require-scripts": {
-      describe: "Execute ./scripts/prepublish.js and ./scripts/postpublish.js, relative to package root.",
-      type: "boolean",
-    },
     "git-reset": {
       describe: "Reset the git working tree.\nPass --no-git-reset to disable.",
       type: "boolean",
       defaultDescription: "true",
     },
     "temp-tag": {
-      describe: "Create a temporary tag while publishing.",
+      describe: "Create a temporary dist-tag while publishing.\nPass --no-temp-tag to disable.",
       type: "boolean",
+      defaultDescription: "true",
```

Comment:
How does this change affect the `lerna help` output? Should it actually default to `no-temp-tag` instead of `temp-tag`?

Reply:
Also, isn't this capable of using a different string here?

Reply:
We're using yargs' "auto-negation" feature here (any boolean option prefixed with `no-` is set to `false`), so it is a bit awkward, I agree. We can't use `"no-temp-tag"` as the key, otherwise it would need to be passed as `--no-no-temp-tag`.

(I could be wrong, it's been awhile since I tried to make this clearer. It could also require upstream changes in yargs)